src/documents/management/commands/document_retagger.py

```diff
@@ -63,6 +63,12 @@ def add_arguments(self, parser):
             "--base-url",
             help="The base URL to use to build the link to the documents.",
         )
+        parser.add_argument(
+            "--id-range",
+            help="A range of document id's on which the retagging should be applied.",
```

Comment:
I just wanted to let you know that there is no need for an apostrophe here :)

Reply:
Hah^^ 
I think it´s a common mistake, especially for non native english speakers :D